Volha_Ivanova/Module_4_Framework/Layers_Runner_BusinessObjects/my-app/src/main/java/com/epam/cdp/volha_ivanova/task1/pages/EmailListPage.java

```diff
@@ -0,0 +1,95 @@
+package com.epam.cdp.volha_ivanova.task1.pages;
```

Comment:
I recommend refactor page classes that contains a lot of elements. You can create classes that correspond to some area of page (set of fields, cards and so on). The classes will contains appropriate elements, and methods
So after that inject the classes to appropriate page. It makes you classes simpler.